cmake-modules/FolderList.cmake

```diff
@@ -2,16 +2,16 @@ macro(GetFolderList project)
     message ("project found ${project}")
     message ("FLAG VALUE : ${FETCH_SOURCE_DEPS}")
     if(${project} STREQUAL CERTIFICATES)
-        DownloadDepVersion(sdk/core azure-core 1.2.0)
-        DownloadDepVersion(sdk/identity azure-identity 1.1.0)
+        DownloadDepVersion(sdk/core azure-core 1.5.0)
+        DownloadDepVersion(sdk/identity azure-identity 1.2.0)
     elseif(${project} STREQUAL IDENTITY)
         DownloadDepVersion(sdk/core azure-core 1.2.0)
     elseif(${project} STREQUAL SECRETS)
-        DownloadDepVersion(sdk/core azure-core 1.2.0)
```

Comment:
What's the purpose of this file? Is this something that should be updated for all SDKs? If so, should Template and Attestation be listed?

Reply:
This file provides the foundation for `FETCH_SOURCE_DEPS` option.  People can use that option when building one only package and cmake would fetch the AzureSDK dependencies using this file configuration

Reply:
Should this be updated for all SDKs? Or just the set listed?

Reply:
we can add other SDKs, as wanted